libs/net/dal/Services/ContentService.cs

```diff
@@ -179,7 +179,7 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindFr
     {
         var productQueries = new List<Func<QueryContainerDescriptor<API.Areas.Services.Models.Content.ContentModel>, QueryContainer>>();
         var today = DateTime.Today.ToUniversalTime();
-        productQueries.Add(q => q.Raw(@"{""match"": {""productId"": 11}}"));
+        productQueries.Add(q => q.Raw(@"{""match"": {""productId"": 10}}"));
```

Comment:
We can't hardcode this.  It shouldn't have been hardcoded in the first place.
Additionally, we don't want searches to use NEST going forward.
We have a way to configure setting values that are essentially constants for each environment.
Talk to @jdtoombs about using an Elasticsearch query from the web app, and how to use Settings.

Reply:
That's nice. I'll work on this.